eng/targets/VisualStudio.targets

```diff
@@ -139,5 +139,14 @@
     <PackageReference Include="Microsoft.DiaSymReader.Native" Version="$(MicrosoftDiaSymReaderNativeVersion)" ExcludeAssets="all"/>
   </ItemGroup>
 
+  <Target Name="_CheckDeploymentTargetVisualStudioVersion"
+		      Condition="'$(DeployExtension)' == 'true' and '$(CreateVsixContainer)' == 'true'"
+          BeforeTargets="DeployVsixExtensionFiles"
+	       	DependsOnTargets="GetVsixDeploymentPath">
+
+    <Error Text="Deployment to Dev17 is not supported in this branch. Use the corresponding *-vs-deps branch." 
```

Comment:
Should this message be adjusted to something like "use the non *-vs-deps branch"?

Reply:
Yes.